tests/utils/nnnunet_utils_test.py

```diff
@@ -0,0 +1,45 @@
+import logging
+import re
+
+import pytest
+from torch.optim import SGD
+
+from fl4health.utils.nnunet_utils import PolyLRSchedulerWrapper
+from tests.test_utils.models_for_test import MnistNetWithBnAndFrozen
+
+
+def test_poly_lr_scheduler(caplog: pytest.LogCaptureFixture) -> None:
+    max_steps = 1000
+    exponent = 1
+    steps_per_lr = 100
+    initial_lr = 0.5
+
+    model = MnistNetWithBnAndFrozen()
+    opt = SGD(model.parameters(), lr=initial_lr)
+    lr_scheduler = PolyLRSchedulerWrapper(
+        optimizer=opt, max_steps=max_steps, initial_lr=initial_lr, exponent=exponent, steps_per_lr=steps_per_lr
+    )
+
+    assert lr_scheduler.num_windows == round(max_steps / steps_per_lr)
```

Comment:
Just a suggestion, but if the calculation is simple, I tend to inject it explicitly here rather than sort of duplicating the code just in case my code is somehow hiding something wrong. That is, here I would just do
```
assert lr_scheduler.num_windows == 10.0
```